docs/how-payjoin-saves-exchanges.md

```diff
@@ -0,0 +1,117 @@
+
+## Payjoin Payment Batching
+
+What if Dave the depositor can payjoin? He gets a benefit of breaking some privacy heuristics, and he can save the exchange some fees that might be able to be pased on to him. Payjoin lets the exchange fund withdrawals with Dave's deposit in the same transaction.
+
+```
+Dave     3 btc  ->  1 btc to Alice
+Exchange 2 btc      1 btc to Bob
+                    1 btc to Carol
+                   ~2 BTC minus fees to Exchange
+```
+
+𝑏 + 2𝑖 + 4𝑜
+
+Payjoin can save more than 1/3 of the cost of making normal transactions by batching them together.
+
+Payjoin lets a sender and receiver combine their transactions when they use Bitcoin. This doesn't just save money. It fixes the biggest privacy problem the Bitcoin network has at the same time.
+
+Imagine Alice would like to send some sats to Bob. Before Payjoin, she'd make a transaction that looks like this:
+
+```
+Alice 500,000 sats  ->  200,000 sats to Bob
+                        292,387 sats to Alice
+
+(7,613 sats are fees for 152.25 vB at 50 sat/vB)
+```
+
+To understand the transaction, know that Bitcoin transactions spend stacks of bitcoin called [unspent transaction outputs (UTXOs)](https://unchained.com/blog/what-is-a-utxo-bitcoin/) to make new transaction outputs. To hold some bitcoin is to control a UTXO. UTXOs have their own denominations like banknotes, but they can hold any amount.
+
+So Alice makes one new UTXO to Bob, and another that holds the change to herself, since UTXOs must be spent completely. The difference between the inputs and outputs is paid to the miners as fees.
+
+Before he gets paid by Alice, Bob knows he wants to consolidate the bitcoin from Alice with other bitcoin to Charlie, perhaps to exchange for local currency.
+
+```
+Bob 200,000 sats ->  491,575 sats to Charlie
+Bob 300,000 sats 
+
+(8,425 sats are fees for 168.5 vB at 50 sat/vB)
+```
+
+Space in the blockchain is limited, and transactions pay for base costs of the transaction header, costs for each input and costs for each output. Making these two transactions costs:
+
+*2 × base costs + 3 × input costs + 3 × output costs*
+
+If Bob knows he needs to pay Charlie, he can Payjoin with Alice to save loads of fees.
+
+Their Payjoin looks like this
+
+```
+Alice 500,000 sats ->  500,000 sats to Charlie
+Bob   300,000 sats     289,412 sats to Alice
+
+(10,588 sats are fees for 211.25 vB at 50 sat/vB)
+```
+
+Since Alice's payment to Bob "[cut-through](https://bitcointalk.org/index.php?topic=281848.0)" to Charlie,
```

Comment:
Question: this feels obvious and dumb to ask, but cut-through is only applicable to unconfirmed transactions, yes? Meaning whatever cut-through the participating parties do must be done opportunistically and any software that uses this would have to have some rules to know when such opportunities are available. This is all just to say we don't really see this ever really being used as a manual process, correct?